### PR TITLE
Added support for embedded lyrics

### PR DESCRIPTION
**Pull request by abelmul**
You can now see an embedded lyrics in mp3 files inside ncmpcpp. It doesn't save the lyrics in the lyrics folder. It is done according to this question https://stackoverflow.com/questions/11051055/how-to-get-the-lyrics-of-a-song-from-the-song-file

### CODE REVIEW

File: src/lyrics_fetcher.cpp [hunk 1]
```diff
@@ -146,6 +152,32 @@ void LyricsFetcher::postProcess(std::string &data) const
...
 /***********************************************************************/
 
+LyricsFetcher::Result FileLyricFetcher::fetch(const std::string &filepath)
+{
+	Result result;
+	result.first = false;
+
+	TagLib::MPEG::File file(filepath.c_str());
+	if(!file.isOpen()){
+		result.second = "Can't open file" + filepath;
```
Comment (tansly) [inline src/lyrics_fetcher.cpp]: There is a missing return in this case. If the file cannot be opened, it goes on to operate on it and it segfaults.
See: [https://github.com/tansly/ncmpcpp/commit/3f048452b79432e6f0a57b470814b20426bcb162]()

**Comment by abelmul** [comment]
Sorry I have seen that and already done it in [here](https://github.com/abelmul/ncmpcpp/blob/master/src/lyrics_fetcher.cpp). Now how can I update this pull request.

File: src/lyrics_fetcher.cpp (hunk 1 above)
Comment (abelmul) [reply src/lyrics_fetcher.cpp]: Sorry I have seen that and already done it in [here](https://github.com/abelmul/ncmpcpp/blob/master/src/lyrics_fetcher.cpp). Now how can I update this pull request.

**Comment by larson-carter** [comment]
I'm going to close this PR due to issues mentioned by @tansly Also @abelmul Unicode is not supported so we will have to find a work around this.

**Pull request closed without merging**